biosys/apps/main/fixtures/test-datasets.json

```diff
@@ -52,14 +52,20 @@
                   "type": "string",
                   "description": "",
                   "name": "Genus",
-                  "title": ""
+                  "title": "",
```

Comment:
I thought these test fixtures were on their way out?

Reply:
They are but slowly. Still used by some tests. I've refactored the species_observations tests to not use them.